wetterdienst/service.py

```diff
@@ -67,40 +68,75 @@ def robots():
     """.strip()
 
 
-@app.get("/api/dwd/stations")
-def dwd_stations(
+@app.get("/api/dwd/{product}/sites")
+def dwd_sites(
+    product: str,
     parameter: str = Query(default=None),
     resolution: str = Query(default=None),
     period: str = Query(default=None),
+    lon: float = Query(default=None),
+    lat: float = Query(default=None),
```

Comment:
Can we also use long names here and add short names as aliases? Also, maybe swap the lines?
```suggestion
    latitude: float = Query(default=None, alias="lat"),
    longitude: float = Query(default=None, alias="lon"),
```